pandas/_libs/window.pyx

```diff
@@ -1391,7 +1392,17 @@ def roll_quantile(ndarray[float64_t, cast=True] input, int64_t win,
 
         if nobs >= minp:
             idx = int(quantile * <double>(nobs - 1))
-            output[i] = skiplist.get(idx)
+
+            # Single value in skip list
+            if nobs == 1:
+                output[i] = skiplist.get(0)
+
+            # Interpolated quantile
+            else:
+                vlow = skiplist.get(idx)
+                vhigh = skiplist.get(idx + 1)
+                output[i] = vlow + (vhigh - vlow) * \
```

Comment:
Very minor nitpick: there seems to be no other occurrence of `\` anywhere in this file. Can someone more familiar with Pandas style requirements comment on whether this is OK, or implicit line continuation inside parentheses is preferred?